packages/@glimmerx/storybook/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@glimmerx/storybook",
+  "version": "0.0.1",
+  "description": "storybook for glimmerjs",
+  "keywords": [
+    "storybook",
+    "glimmerjs"
+  ],
+  "author": "Jonathan Tsang <jonathantsangemail@gmail.com>",
+  "license": "MIT",
+  "files": [
+    "bin/**/*",
+    "dist/**/*",
+    "README.md",
+    "*.js"
+  ],
+  "main": "dist/client/index.js",
+  "bin": {
+    "build-storybook": "./bin/build.js",
+    "start-storybook": "./bin/index.js",
+    "storybook-server": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "node ./scripts/prepare.js",
+    "test": "jest && yarn lint",
```

Comment:
Please use qunit for testing in this repo, since that is what is being used in the other packages.

Reply:
done!